presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -96,6 +96,125 @@ Finally, you can access the ``clicks`` table in the ``web`` database::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
 
+Type mapping
+------------
+
+Because PrestoDB and MySQL each support types that the other does not. When reading from or writing to MySQL,
+Presto will convert the data types from MySQL to equivalent Presto data types and vice versa.
```

Comment:
```suggestion
Presto converts the data types from MySQL to equivalent Presto data types, and from Presto to equivalent MySQL data types.
```
Vice versa is a Latin phrase that should not be used in documentation. The [GitLab word list](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html) recommends "Do not use Latin abbreviations.", but does not have a specific entry or advice for avoiding Latin phrases such as vice versa. 

The [Google documentation style guide has an entry for vice versa](https://developers.google.com/style/word-list#vice-versa) and suggests "In such cases, make it explicitly clear what two things are swapped." 

My suggestion here attempts to be clear what two things are swapped. 
If I have changed the meaning in a way that you disagree with, I'm happy to discuss. 

Separate change: "will" is future tense and I'm suggesting a change to present tense, "Presto will convert" becomes "Presto converts".